assets/src/blocks/Timeline/Timeline.js

```diff
@@ -35,8 +35,12 @@ export const Timeline = (props) => {
 
 	useEffect(
 		() => {
-			if (stylesLoaded && scriptLoaded) {
-				setupTimeline();
+			if (scriptLoaded) {
+				_.noConflict();
```

Comment:
We're only using lodash to generate a unique ID, maybe we can generate the ID another way and remove the dependency?

Reply:
From what I've tested it's not our usage (tried removing uniqueId and debounce), it's the timeline lib itself that is the problem https://cdn.knightlab.com/libs/timeline3/3.8.10/js/timeline.js

Reply:
Also tried compiling their lib with different options to restrict lodash import to some functions, but I'm not very competent :grimacing:

Reply:
Right, missed that by not reading your description well enough :facepalm: Is there a way we can prevent the timeline JS from overriding `_`? If we run `_.noConflict` after it there will still be a window in which `wp` could crash if it's trying to use underscore. Though we could have a look at how WP uses underscore, maybe there's just a few usages and they're probably not going to use it more extensively I guess.

Reply:
The window would be quite small, so I guess this fix is good. Maybe it's better to create a separate effect with just the `scriptLoaded` dependency that only does the noConflict?

Reply:
> Is there a way we can prevent the timeline JS from overriding `_`?

Couldn't find a local solution; so  I looked at compiling their lib with https://github.com/lodash/babel-plugin-lodash, it's supposed to pick only the used functions of lodash without the complete `_` object - but I think I'm failing to use it properly (my compiled file is exactly the same), so if anybody with some experience at this could try it that'd be helpful :pray:  
I don't see any other solution at the moment, but I'll take any idea :bulb: